src/main/java/org/junitpioneer/jupiter/cartesian/CartesianTest.java

```diff
@@ -85,7 +84,7 @@
 
 	@Retention(RetentionPolicy.RUNTIME)
 	@Target({ ElementType.PARAMETER, ElementType.ANNOTATION_TYPE })
-	@ArgumentsSource(CartesianValueArgumentsProvider.class)
+	@CartesianArgumentsSource(CartesianValueArgumentsProvider.class)
```

Comment:
No longer being usable with `@ArgumentsSource` seems like a good thing, too, because these providers couldn't work with "just" the parameterized machinery anyway and so this removes a potential mistake that the user can make.

Reply:
Actually it is usable with both. e.g. the different `XXXRangeSource` annotations only define `@ArgumentsSource`. However, the `RangeSourceArgumentsProvider` implements the `CartesianArgumentsProvider`. 

However, with the current approach you can entirely opt in to only `@CartesianTest` by using the `@CartesianArgumentsSource`. This way users won't use your `@CartesianTest` annotation in `@ParametrizedTest`